6.2.running_2000.py

```diff
@@ -0,0 +1,26 @@
+from time import time
+
+
+def timer(f, *args, **kwargs):
+    """
+      A function that times the execution of a given function and returns the time taken.
+
+      Args:
+      f (function): A function to be timed.
+      *args: Any number of positional arguments to be passed to the function.
+      **kwargs: Any number of keyword arguments to be passed to the function.
+
+      Returns:
+      the_final_time (float): The time taken to execute the given function.
+      """
+    start = time()
+    f(*args, **kwargs)
+    finish = time()
+    the_final_time = finish - start
+    return the_final_time
+
+
+#a = timer(print, "Hello")

```

Comment:
Do not leave documented code in your script. it's better to put it in a test function.